recipe/meta.yaml

```diff
@@ -90,6 +91,7 @@ outputs:
         - pyarrow <13.0 # [osx]
         - pyarrow       # [not osx]
         - pyarrow-hotfix
+        - libgoogle-cloud 2.12.*
```

Comment:
Isn't this supposed to be 2.21?

Reply:
(actually, why are we pinning this at all?)

Reply:
It is not

See Slack

Reply:
I'll add some details here momentarily

Reply:
> (actually, why are we pinning this at all?)

It's a hacky workaround we are applying to unblock tiledbsoma. Originally discovered in #89, this solver error came out of nowhere. I suspect it has to do with the recent conda-forge switch from google-cloud-cpp to libgoogle-cloud, but I've been too busy with the libcurl 7 stuff to figure out exactly what is going on.

Reply:
Ok, carry on - thanks.

Reply:
For this PR 2.12 doesn't work although it did for #93:
https://gist.github.com/johnkerl/7812fb057db778d8bfb17b424dec6ead

Trying 2.17 locally ... 🤞

Reply:
libgoogle-cloud 2.17 fixed the conda solver error. However, only the linux-64 build is green